src/AltinnRepositories/gitea-data/gitea/templates/base/head.tmpl

```diff
@@ -141,23 +141,27 @@
 			<div class="ui top secondary stackable main menu following bar light">
 				<div class="ui container" id="navbar">
 					<div class="item brand" style="justify-content: space-between;">
-						<a href="{{AppSubUrl}}/" href="{{AppSubUrl}}/">
-							<img class="ui mini image" src="{{AppSubUrl}}/img/altinn_logo_header.png">
+						<a href="/" title="Altinn Studio {{.i18n.Tr "home"}}">
+							<img class="ui mini image" src="{{AppSubUrl}}/img/favicon.png" alt="Altinn logo">
 						</a>
 						<div class="ui basic icon button mobile-only" id="navbar-expand-toggle">
 							<i class="sidebar icon"></i>
 						</div>
 					</div>
 
 					{{if .IsSigned}}
-						<a class="item{{if .PageIsDashboard}} active{{end}}" href="{{AppSubUrl}}/">{{.i18n.Tr "dashboard"}}</a>
+            <a class="item{{if .PageIsDashboard}} active{{end}}" href="{{AppSubUrl}}/">{{.i18n.Tr "dashboard"}}</a>
```

Comment:
Should we change this to href to "/", so that it navigates to studio dashboard?

Reply:
No, this is the link to the Gitea dashboard (now working). The link to Studio Dashboard is still the logo to the left in the menu.